BZ-1870931 adding cluster logging must-gather

lbarbeevargas opened the pull request:
BZ-1870931 https://bugzilla.redhat.com/show_bug.cgi?id=1870931

@nekop PTAL. I wasn't sure if the cluster logging must-gather needed to be added for openshift-origin as well. Did I add the information in the list that you were referring to in the BZ? Also, I looked up the Cluster Logging Operator image on `registry.redhat.io` for the openshift-origin example.

lbarbeevargas commented:
> BZ-1870931 https://bugzilla.redhat.com/show_bug.cgi?id=1870931
> 
> @nekop PTAL. I have a few questions as well:
> 
> * I wasn't sure if the cluster logging must-gather needed to be added for openshift-origin as well. Did I add the information in the list that you were referring to in the BZ?
> * Also, I looked up the Cluster Logging Operator image on `registry.redhat.io` for the openshift-origin example. Did I use the correct image?
> * What versions of the docs does this information need to be added? 4.4+ ?

@nekop PTAL, thank you!

nekop commented:
> * I wasn't sure if the cluster logging must-gather needed to be added for openshift-origin as well. Did I add the information in the list that you were referring to in the BZ?
> * Also, I looked up the Cluster Logging Operator image on `registry.redhat.io` for the openshift-origin example. Did I use the correct image?

I have no idea on the openshift-origin side.

> * What versions of the docs does this information need to be added? 4.4+ ?

4.4+ would be ok.

lbarbeevargas commented:
@nekop PTAL, thank you.

I created a separate section specifically for "Gathering cluster logging data" - [Preview](https://bz-1870931-cluster-logging-must-gather--ocpdocs.netlify.app/openshift-enterprise/latest/support/gathering-cluster-data.html#gathering-cluster-logging-data_gathering-cluster-data).

nekop commented:
LGTM

lbarbeevargas commented:
@anpingli @openshift/team-documentation PTAL, thank you!

adellape commented on modules/cluster-logging-must-gather.adoc:
```diff
@@ -0,0 +1,170 @@
+
+.Prerequisites
+
+* Access to the cluster as a user with the cluster-admin role.
```
`cluster-admin`

adellape commented on modules/cluster-logging-must-gather.adoc:
```diff
@@ -0,0 +1,170 @@
+
+* Access to the cluster as a user with the cluster-admin role.
+
+* The OpenShift Container Platform CLI (oc) installed.
```
`oc`

adellape commented on modules/cluster-logging-must-gather.adoc:
```diff
@@ -0,0 +1,170 @@
+.Procedure
+
+* Run the following `must-gather` command to gather cluster logging data:
+
```
Needs a `+`

adellape commented on modules/cluster-logging-must-gather.adoc:
```diff
@@ -0,0 +1,170 @@
+----
+oc adm must-gather --image=quay.io/openshift/origin-cluster-logging-operator -- /usr/bin/gather
+----
+
```
Another `+`

adellape commented on modules/cluster-logging-must-gather.adoc:
```diff
@@ -0,0 +1,170 @@
+
+[source,terminal]
+----
+oc adm must-gather --image=quay.io/openshift/origin-cluster-logging-operator -- /usr/bin/gather
```
`$` prompt

lbarbeevargas replied:
Also updating the command per feedback from @mburke5678.

adellape commented on modules/cluster-logging-must-gather.adoc:
```diff
@@ -0,0 +1,170 @@
+[id="gathering-cluster-logging-data_{context}"]
+= Gathering cluster logging data
+
+You can use the `cluster-logging-must-gather` tool to gather data that is specifically related to cluster logging and the Cluster Logging Operator in your {product-title} cluster.
```
Is there actually a tool (or maybe some component) called `cluster-logging-must-gather`? I couldn't find one from poking around a little, and the command in the procedure is just using `oc adm must-gather`. Is it more just something like, "You can use the `must-gather` tool with additional arguments to gather data..."?

lbarbeevargas replied:
Updated wording per feedback because this is built on top of the `must-gather` tool.

anpingli reviewed (commented):
/lgtm

lbarbeevargas commented:
@nekop @anpingli -- I updated the `$ oc must-gather` command after receiving peer review feedback and feedback from the Logging team writer. Can you PTAL at the changes, especially the command? Thank you!

lbarbeevargas commented:
> @nekop -- I updated the `$ oc must-gather` command after receiving peer review feedback and feedback from the Logging team writer. Can you PTAL at the changes, especially the command? Thank you!

@nekop Bump. Thanks!

nekop commented:
LGTM

lbarbeevargas commented:
**Update note:** Michael Burke and I were simultaneously working on this same info. I am building on the cluster logging `must-gather` info that Michael already merged to add an intro and example output for cluster logging `must-gather`. Also, I removed a new module that I originally created for this PR and just added the info to the existing `gathering-data-specific-features.adoc` module.

kalexand-rh reviewed (commented):
LGTM!

lbarbeevargas commented:
/cherrypick enterprise-4.4

lbarbeevargas commented:
/cherrypick enterprise-4.5

lbarbeevargas commented:
/cherrypick enterprise-4.6

lbarbeevargas commented:
/cherrypick enterprise-4.7
